db/fixtures/miq_product_features.yml

```diff
@@ -3971,6 +3971,10 @@
       :description: Edit Tags of Middleware Domains
       :feature_type: control
       :identifier: middleware_domain_tag
+    - :name: Stop Middleware Domain
+      :description: Stop Middleware Domain
+      :feature_type: admin
```

Comment:
@josejulio normally, `start` and `stop` type operations are given a feature type of `control`.  We use `admin` for CRUD type features.  So, I'd think you would want this as a `control` feature type.